### PR TITLE
windows: navigator build

### PR DESCRIPTION
**Pull request by majcherm-da**
- fixes Webpack's paths format issue (`/c/` ... `is not an absolute path!` issue ) on Windows
- fixes `jar` / `jar.exe` binary resolution
- adds `//navigator/backend/...` and `//navigator/frontend/...` to Win CI

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/docs/source/support/release-notes.rst), if appropriate

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -26,13 +26,16 @@ function build-partial() {
         //:git-revision `
         //compiler/daml-lf-ast/... `
         //daml-lf/interface/... `
-        //language-support/java/bindings/...
+        //language-support/java/bindings/... `
+        //navigator/backend/... `
```
Comment (moritzkiefer-da) [inline build.ps1]: Is there anything in `//navigator` that doesn’t build yet? If not we could just use `//navigator/...`.

**Comment by stefanobaghino-da** [comment]
I believe @rautenrieth-da should be a reviewer here, rather than me.

File: build.ps1 (hunk 1 above)
Comment (majcherm-da) [reply build.ps1]: Yes, integration-tests reference `damlc-dist`

**Comment by majcherm-da** [comment]
On CI `_bazel_VssAdministrator` is mixed case and causes `CaseSensitivePathsPlugin` to throw an error:
`C:\users\vssadministrator\_bazel_vssadministrator\w3d6ug6o\external\navigator_frontend_deps\node_modules\process\browser.js` does not match the corresponding path on disk `_bazel_VssAdministrator`. 

I'm looking for a solution now.

File: build.ps1 (hunk 1 above)
Comment (rautenrieth-da) [reply build.ps1]: If it helps, the integration test needs something to compile DAML to DAR, but we can change how exactly this is achieved.
I see:
- https://github.com/digital-asset/daml/blob/master/navigator/integration-test/src/main/scala/com/digitalasset/navigator/test/runner/PackagedDamlc.scala, using `com.digitalasset.damlc.Runner`
- https://github.com/digital-asset/daml/blob/master/navigator/integration-test/src/main/scala/com/digitalasset/navigator/test/runner/HeadDamli.scala, using `bazel run damlc`
- I think I saw I some other project that compiled DAML to DAR at compile time, and included the DAR file as a resource.

The different files at `navigator/test/runner` are from before we had head based development and should be cleaned up.

**Comment by rautenrieth-da** [comment]
CaseSensitivePathsPlugin is optional, and meant to catch errors coming from case sensitivity of file systems. We can also remove it if we test the build on all supported systems.

E.g., `_bazel_VssAdministrator` and `_bazel_vssadministrator` refer to the same file on Windows or MacOS. If we only tested on these systems, we would not realize that the same build is broken on Unix, where these two strings refer to different files.

**Comment by majcherm-da** [comment]
@rautenrieth-da I disabled `CaseSensitivePathsPlugin` when running on Windows (from Bazel only) as sth. gets paths lowercased and I couldn't find an easy way to get proper cased paths on Windows when in genrule's bash script. `CaseSensitivePathsPlugin` is still applied on Linux & Mac.

**Pull request merged**